packages/expo-image/android/src/main/java/expo/modules/image/ExpoImageView.kt

```diff
@@ -50,6 +50,7 @@ class ExpoImageView(context: ReactContext, private val requestManager: RequestMa
   }
   private var loadedSource: GlideUrl? = null
   internal var sourceMap: ReadableMap? = null
+  private var fadeDuration: Int? = null
```

Comment:
```suggestion
  var fadeDuration: Int? = null
    private get
```
I would rather go with [kotlin properties here](https://kotlinlang.org/docs/properties.html#getters-and-setters)